websites/ribbons/index.html

```
@@ -110,69 +110,139 @@
-          <path
-            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-1 17.93c-3.94-.49-7-3.85-7-7.93s3.05-7.44 7-7.93v15.86zm2 0V4.07c3.94.49 7 3.85 7 7.93s-3.06 7.44-7 7.93z"
-          />
-        </svg>
-      </a>
+      <div class="header-left">
+        <a class="header-link" href="/blog/index.html">Blog</a>
+        <a class="header-link" href="/cookie-policy.html">Cookie</a>
+        <a class="header-link" href="/privacy-policy.html">Privacy</a>
+      </div>
+      <div class="header-right">
+        <button id="themeToggle" title="Switch Theme" class="theme"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)